medusa/init/__init__.py

```diff
@@ -109,7 +110,28 @@ def _strptime_workaround():
 
 
 def _configure_guessit():
-    """Replace guessit function with a pre-configured one, so guessit.guessit() could be called directly in any place."""
+    """Replace guessit with a pre-configured one, so guessit.guessit() could be called directly in any place."""
     import guessit
     from ..name_parser.guessit_parser import guessit as pre_configured_guessit
     guessit.guessit = pre_configured_guessit
+
+
+def _configure_subliminal():
+    """Load subliminal with our custom configuration."""
```

Comment:
@ratoaq2 can we override `SHOW_EXPIRATION_TIME` here from 3 weeks to 3 days or else?